### PR TITLE
fix apache revision

### PR DESCRIPTION
**Pull request by z7658329**
fix apache revision

### CODE REVIEW

**Comment by z7658329** [comment]
the apache stage version, still have some problems:
https://repository.apache.org/content/groups/staging/org/apache/hugegraph/hugegraph-common/1.0.0/hugegraph-common-1.0.0.pom

it will cause hugegraph-client build error:
image

image

i fix this problem in local:
image

image

**Comment by z7658329** [comment]
ut have some error, but test in local is ok
image

**Comment by z7658329** [comment]
@javeme @imbajin 
test in local is ok
image

failed in ci, see:
https://github.com/apache/incubator-hugegraph-commons/actions/runs/3390737617/jobs/5635202250

image

File: pom.xml [hunk 1]
```diff
@@ -217,6 +218,32 @@
                     <consoleOutput>true</consoleOutput>
                 </configuration>
             </plugin>
+
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>flatten-maven-plugin</artifactId>
+                <version>1.2.7</version>
```
Comment (imbajin) [inline pom.xml:225]: any extra cmd do we need?

**Review by javeme — APPROVED** [review]

**Pull request merged**